feature/experimental/p4rt/otg_tests/traceroute_packetin_test/packetin_test.go

```diff
@@ -140,7 +142,13 @@ func testPacketIn(ctx context.Context, t *testing.T, args *testArgs, isIPv4 bool
 
 	// Send Traceroute traffic from ATE
 	srcEndPoint := ateInterface(t, args.top, "port1")
-	dstMac := gnmi.Get(t, args.ate.OTG(), gnmi.OTG().Interface(atePort1.Name+".Eth").Ipv4Neighbor(dutPort1.IPv4).LinkLayerAddress().State())
+	llAddress, found := gnmi.Watch(t, args.ate.OTG(), gnmi.OTG().Interface(atePort1.Name+".Eth").Ipv4Neighbor(dutPort1.IPv4).LinkLayerAddress().State(), time.Minute, func(val *ygnmi.Value[string]) bool {
+		return val.IsPresent()
+	}).Await(t)
+	if !found {
+		t.Errorf("Could not get the LinkLayerAddress %s", llAddress)
```

Comment:
Should this be t.Fatalf? We want that if the test can't continue correctly without the LinkLayerAddress

Reply:
yes..makes sense, changed to Fatalf